From 621b86cda6b00c99b3ce9d31a0ab9dcb1905ccaf Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 26 Feb 2019 20:46:54 +0100
Subject: [PATCH 1/1] cmd: set CONFIG_SYS_HELP_CMD_WIDTH = 10

CONFIG_SYS_HELP_CMD_WIDTH is used to format the output of help without any
arguments.

CONFIG_SYS_HELP_CMD_WIDTH = 8 is too narrow to fit all our commands.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/command.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/command.h b/include/command.h
index 2e24e8ad3ee..be74f6ac92f 100644
--- a/include/command.h
+++ b/include/command.h
@@ -18,7 +18,7 @@
 
 /* Default to a width of 8 characters for help message command width */
 #ifndef CONFIG_SYS_HELP_CMD_WIDTH
-#define CONFIG_SYS_HELP_CMD_WIDTH	8
+#define CONFIG_SYS_HELP_CMD_WIDTH	10
 #endif
 
 #ifndef	__ASSEMBLY__
-- 
2.20.1

